packages/near-api-js/test/account_multisig.test.js

```diff
@@ -131,5 +145,99 @@ describe('account2fa transactions', () => {
         const state = await receiver.state();
         expect(BigInt(state.amount)).toBeGreaterThanOrEqual(BigInt(new BN(receiverAmount).add(new BN(parseNearAmount('0.9'))).toString()));
     });
-        
+
+});
+
+describe('2fa batch key conversion', () => {
+    let sender;
+    beforeAll(async () => {
+        sender = await testUtils.createAccount(nearjs);
+        sender = await getAccount2FA(sender);
+    });
+
+    beforeEach(async () => {
+        sender = await getAccount2FA(sender, undefined, false);
```

Comment:
This ended up being a much faster way to run these tests than creating an account per test as above. The `beforeAll` block creates the account and deploys the contract while the `beforeEach` only resets the instance between tests to retain isolation.